stable/prometheus/templates/server-statefulset.yaml

```diff
@@ -183,3 +176,28 @@ spec:
           configMap:
             name: {{ .configMap }}
       {{- end }}
+{{- if .Values.server.persistentVolume.enabled }}
+  volumeClaimTemplates:
+    - metadata:
+        name: storage-volume
```

Comment:
Not sure - but will the PVC/PV be named storage-volume-<replicaNum>? - would it be more informative to call it prometheus-volume or something similar? 

In the elastic chart it's templated in values.yaml: https://github.com/helm/charts/blob/master/incubator/elasticsearch/values.yaml

Reply:
No, it will be called something like `storage-volume-RELEASE-NAME-prometheus-server-0`.